src/components/CourseCard/BaseCourseCard.jsx

```diff
@@ -15,7 +15,7 @@ class BaseCourseCard extends Component {
     modals: {
       emailSettings: null,
     },
-    hasEmailsEnabled: this.props.hasEmailSettings || false,
+    hasEmailsEnabled: this.props.hasEmailsEnabled,
```

Comment:
This fixes a bug I noticed. The `hasEmailSettings` prop is meant for whether the "Email Settings" button shows up in the course card, whereas `hasEmailsEnabled` is for learners' actual email setting for each course.